src/hooks/useJoystream/provider.tsx

```diff
@@ -11,6 +11,7 @@ type JoystreamContextValue = {
   joystream: JoystreamJs | null
   accounts: Account[]
   extensionConnected: boolean
+  extensionConnectionLoading: boolean
```

Comment:
Could we possibly reuse `extensionConnected` for that? It could have 3rd intermediary step, for example `null`. What do you think about that?